README.md

```diff
@@ -1,2 +1,13 @@
-# content-sources-backend
-Backend for content sources service
+# Content Sources
+
+##What is it?
+Content Sources is an application for storing content references (currently YUM repositories) in a central location.   
```

Comment:
It might be good to define what a "content reference" is, maybe with an example.